doc/ug/getting_started_fpga.md

```diff
@@ -36,13 +36,24 @@ $ ./meson_init.sh
 $ ninja -C build-out sw/device/boot_rom/boot_rom_export_fpga_nexysvideo
 ```
 
+Since not all FPGAs are able to fit the full design, there is a separate script that can be invoked to reduce the size of the design.
+
+To reduce the design:
+```console
+$ cd $REPO_TOP
+$ ./hw/top_earlgrey/util/opentitan_earlgrey_flash_size_reduce.py
+```
+By default, the reduce script targets 'nexysvideo', cw305 is also supported.
```

Comment:
You can drop `cw305`, earlgrey is anyway far too big for the small FPGA and the CW305. We will have to remove those legacy files.